ingest/defaults/sublineage_metadata.tsv

```diff
@@ -203,10 +203,10 @@ MK117952	3e
 MK117989	3e
 MK118011	3e
 MK118034	3e
-MK107927	a
-HM143866	b
-JN651370	b
-KT992436	c
-KT992437	c
-KT992438	c
-KT992439	c
\ No newline at end of file
+MK107927	1a
+HM143866	1b
+JN651370	1b
+KT992436	1c
+KT992437	1c
+KT992438	1c
+KT992439	1c
```

Comment:
@JoiRichi and I chatted on [slack](https://bedfordlab.slack.com/archives/C08Q64EGTTP/p1750287616903959?thread_ts=1750263287.084449&cid=C08Q64EGTTP) about some ambiguity in the [paper](https://doi.org/10.1128/jvi.00929-19) on if these are supposed to be classified as 1a,b,c (based on supplemental) or 6a,b,c (based on tree in figure 1). We decided to leave it as a,b,c since we were unsure which part was correct for now.